config/settings.yml

```diff
@@ -35,7 +35,12 @@
 
     # disable regions by their keys found in app/models/manageiq/providers/amazon/regions.rb e.g.
     # - us-gov-west-1
-    :disabled_regions: []
+    :disabled_regions:
```

Comment:
@agrare I ran `rake aws:extract:regions` was I supposed to? It's added all the other disabled regions?

Reply:
Hey @juliancheal we don't disable any regions upstream as of 868311b6c5761f1c9f237ecfbba992cd748e0785
Did that rake task add these to the disabled_regions?

Reply:
Ah cool, thanks. Yes it did. Ok I'll remove those lines in this PR. And take a look at the rake task.

Reply:
Awesome, yeah Alex must have forgotten to update the rake task.